src/main/java/de/tum/in/www1/exerciseapp/web/rest/QuizExerciseResource.java

```diff
@@ -51,7 +51,7 @@ public QuizExerciseResource(QuizExerciseRepository quizExerciseRepository, Parti
      * @throws URISyntaxException if the Location URI syntax is incorrect
      */
```

Comment:
please apply the same logic here as for programming exercises

Reply:
Didn't you say that quiz and modeling exercises should be done by the respective authors? I asked you when I wasn't able to map quiz endpoints, and I believed you said something along the lines that the respective authors should do that. Should I then just blindly add checks to all endpoints whether they are used or not?